docs/usage/configuration-options.md

```diff
@@ -1676,7 +1677,10 @@ Any PR that is being updated will be automerged with the Renovate-based automerg
 This option is available for Azure, GitHub and GitLab.
 It falls back to Renovate-based automerge if the platform-native automerge is not available.
 
-Though this option is enabled by default, you can fine tune the behavior by setting `packageRules` if you want to use it selectively (e.g. per-package).
+You can also fine tune the behavior by setting `packageRules` if you want to use it selectively (e.g. per-package).
+
+If you rely on Renovate's behavior of rebasing automerging branches when `rebaseWhen=auto`, then note that you are no longer guaranteed of up-to-date branches when automerging if you enable platform automerge, unless you use some type of branch protection rules to achieve the same.
```

Comment:
This sentence is too long, and needs to be split up or rewritten. It's too hard and understand in one go.